packages/core/src/styles/layout/_display.scss

```diff
@@ -40,18 +40,33 @@
   display: inline !important;
 }
 
-@media (min-width: $cds-token-layout-width-sm-static) {
+[cds-layout~='display:screen-reader-only'] {
+  position: absolute !important;
+  clip: rect(1px, 1px, 1px, 1px);
+  clip-path: inset(50%);
+  padding: 0;
+  border: 0;
+  height: 1px;
+  width: 1px;
+  overflow: hidden;
+  white-space: nowrap;
+  top: 0;
+  left: 0;
+  display: block !important;
+}
```

Comment:
I think this was recently added to the reset? I may of accidentally added this back in. I'll double check.